project/SparkBuild.scala

```diff
@@ -723,8 +723,8 @@ object Unidoc {
         .map(_.filterNot(_.getCanonicalPath.contains("org/apache/hadoop")))
     },
 
-    // Javadoc options: create a window title, and group key packages on index page
-    javacOptions in doc := Seq(
+    // Javadoc options: create a window title
```

Comment:
I think we can either change it to just `// Javadoc options` to clarify that the following `javacOptions` are in fact for Javadoc, or we can remove the comment entirely.